### ISSUE TITLE
ShowcaseState created a Ticker via its TickerProviderStateMixin, but at the time dispose() was called on the mixin, that Ticker was still active

### ISSUE THREAD

**Issue by kmccmk9** [simformsolutions/flutter_showcaseview#6]
Hello,

I've used this pub a few times across an app I'm using with now issue. This is the first time I'm trying to use it without a button interaction. I've tried initiating the showcase from initState, didChangeDependencies and from the build method but I keep getting the following error/exception after I try to Pop the current screen. Note: The showcase does play, but upon disposing the screen (no matter how much time is waited), this error will occur and cause an app crash. Below is the error and stack trace.

```
The following assertion was thrown while finalizing the widget tree:
I/flutter ( 4324): _ShowcaseState#2e4a0(tickers: tracking 1 ticker) was disposed with an active Ticker.
I/flutter ( 4324): _ShowcaseState created a Ticker via its TickerProviderStateMixin, but at the time dispose() was
I/flutter ( 4324): called on the mixin, that Ticker was still active. All Tickers must be disposed before calling
I/flutter ( 4324): super.dispose(). Tickers used by AnimationControllers should be disposed by calling dispose() on the
I/flutter ( 4324): AnimationController itself. Otherwise, the ticker will leak.
I/flutter ( 4324): The offending ticker was: _WidgetTicker(created by _ShowcaseState#2e4a0(lifecycle state: created,
I/flutter ( 4324): tickers: tracking 0 tickers))
I/flutter ( 4324): The stack trace when the _WidgetTicker was actually created was:
I/flutter ( 4324): #0      new Ticker.<anonymous closure> 
package:flutter/…/scheduler/ticker.dart:64
I/flutter ( 4324): #1      new Ticker 
package:flutter/…/scheduler/ticker.dart:66
I/flutter ( 4324): #2      new _WidgetTicker 
package:flutter/…/widgets/ticker_provider.dart:225
I/flutter ( 4324): #3      __ShowcaseState&State&TickerProviderStateMixin.createTicker
I/flutter ( 4324): (package:flutter/src/widgets/ticker_provider.dart:161:34)
I/flutter ( 4324): #4      new AnimationController 
package:flutter/…/animation/animation_controller.dart:245
I/flutter ( 4324): #5      _ShowcaseState.initState 
package:showcaseview/showcase.dart:107
```

**Comment by simformsolutions** [comment]
We're looking into it. Thanks for using and sharing feeedback.

**Comment by simformsolutions** [comment]
Fixed issue in version 0.1.2 , please update package. thanks @ibhavikmakwana
